dlt/dlt.py

```diff
@@ -614,6 +614,9 @@ def __init__(self, **kwords):
         self.end = False
         self.live_run = kwords.pop("is_live", False)
         self.stop_reading = threading.Event()
+        # Shall we allow None returned if no new message coming?
+        # This should not work without self.live_run is True
+        self.yield_none_allowed = kwords.pop("yield_none_allowed", False)
```

Comment:
The variable is not easy to understand. I guess that you want to add non-blocking mode in `__iter__` iteration. Please consider a good variable name to control the state.

Reply:
renamed